i18n/fr.json

```diff
@@ -2184,7 +2184,7 @@
   },
   {
     "id": "api.user.saml.invalid_extension",
-    "translation": "L'extension ayant l'extension_id={{. ExtensionId}} n'est pas supportée."
+    "translation": "L'extension ayant l'extension_id={{.ExtensionId}} n'est pas supportée."
```

Comment:
FYI @wget ^

Reply:
Ok thanks. Very sorry for the issue since the CI failing was due to French. :-/